### PR TITLE
Add org.gabmus.gfeeds

### PR DESCRIPTION
**Pull request by GabMus**
This app is a resubmission of my own `org.gabmus.gnome-feeds`. I changed the appid, thus the new PR.

### CODE REVIEW

**Comment by GabMus** [comment]
bot, build org.gabmus.gfeeds

**Comment by bilelmoussaoui** [comment]
We do support app-id renaming on Flathub now. 
CC @barthalion

**Comment by GabMus** [comment]
How would I go about renaming my app? Any docs you can link?

**Comment by bilelmoussaoui** [comment]
Nothing you can do yourself, at least, not for now. We don't want apps changing their app-ids every time neither.

**Comment by barthalion** [comment]
We were talking about it yesterday on IRC. When this PR is merged, you need to add `end-of-life` and `end-of-life-rebase` to `flathub.json` like I did for [gxi](https://github.com/flathub/com.github.Cogitri.gxi/blob/master/flathub.json) pointing to new app id.

**Comment by GabMus** [comment]
@barthalion then that's exactly what I meant with this PR! I think I'm all set, [the new version with the old appid is ready, along with the flathub.json with the end-of-life stuff](https://gitlab.com/gabmus/gnome-feeds/tree/eol/dist/flatpak).

Once the version with the new appid is out, I'll be pushing the update to the old one.

File: org.gabmus.gfeeds.json [hunk 1]
```diff
@@ -0,0 +1,653 @@
...
+                    "commands" : [
+                        "sed -i 's/if not platform_/if not dirs/' setup.py"
+                    ]
+                }
+            ]
+        },
+        {
+            "name" : "gfeeds",
+            "buildsystem" : "meson",
+            "builddir" : true,
```
Comment (bilelmoussaoui) [inline org.gabmus.gfeeds.json]:
```suggestion
```
already set by default

**Comment by bilelmoussaoui** [comment]
bot, build org.gabmus.gfeeds

**Comment by bilelmoussaoui** [comment]
Repository has been created: https://github.com/flathub/org.gabmus.gfeeds


Thanks!

**Comment by GabMus** [comment]
Will the old app automatically update to the new appid when it gets published?

**Comment by bilelmoussaoui** [comment]
Yes

**Comment by GabMus** [comment]
The version with the new appid is out, but the version with the old appid is still available. Something went wrong?

Screenshot from 2019-09-03 18-44-16

**Pull request closed without merging**